content/docs/quickstart/_index.md

```diff
@@ -1,22 +1,35 @@
 ---
 title: Quick Start
-description: Learn how to get Keptn running in five minutes. Whether you prefer Helm, Docker or Keptn CLI, or k3d, we have you covered.
+description: Familiarize yourself with Keptn by running small exercises on a local Keptn installation
 icon: concepts
 layout: quickstart
 weight: 10
 hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-This Quick Start guide shows how to quickly create a local Keptn installation
-that runs either as a [Helm](#helm) chart in Kubernetes
-or as [Docker](#keptn-hello-world-docker-based) container in K3d
-and run some exercises to demonstrate Keptn functionality.
+This Quick Start guide enables you to quickly access a Keptn environment locally
+and run some simple exercises that demonstrate Keptn functionality.
+The steps are:
 
-You can install and run Keptn on virtually any Kubernetes cluster.
-See Install CLI and Keptn for detailed instructions
-about creating a Keptn cluster locally or in the cloud.
+* Download a container (either Docker or Kubernetes)
+that providea a small but complete Kubernetes cluster with Keptn installed.
+* Run a simple exercise that runs a Quality Gate
+against a multi-stage delivery operation.
+* Run another exercise that shows how the Keptn auto-remediation feature
+can automatically fix problems that the quality gates report.
 
-## Helm
+## Install Keptn in a container
+
+We provide a Helm chart and a Docker container that you can install locally
```

Comment:
We provide a Docker container that you can install locally. Alternatively, we provide a Helm chart that can be used to install on your own cluster...

Reply:
Applied